Gemfile

```diff
@@ -1,15 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'oj'
-gem 'htmlentities'
-gem 'net-dns'
-gem 'bit-struct'
-gem 'geoip-c'
-gem 'maxmind-db', '~> 1.0.0'
-gem 'recog', '~> 3.0'
+gemspec name: 'dap'
 
-group :test do
-  gem 'rspec', '~> 3.9.0'
-  gem 'cucumber', '~> 4.0.0'
-  gem 'aruba', '~> 0.6.2'
+group :development do
+  gem 'yard'
+end
+
```

Comment:
`yard` is now in the gemspec so not needed here.
```suggestion
```

Reply:
looking further at the change all of the `development` gems are now in the spec, if you want to keep the duplication for a skeleton that is reasonable, just not required.